packages/compartment-mapper/src/map-parser.js

```diff
@@ -19,14 +16,17 @@ import { parseExtension } from './extension.js';
  *   ModuleTransform,
  *   ModuleTransforms,
  *   ParseFn,
- *   ParseFnAsync,
+ *   AsyncParseFn,
  *   ParseResult,
  *   ParserForLanguage,
  *   SyncModuleTransform,
  *   SyncModuleTransforms
  * } from './types.js';
  */
 
+import { syncTrampoline, asyncTrampoline } from '@endo/trampoline';
+import { parseExtension } from './extension.js';
```

Comment:
This moved in a fit of homogenization of the order of type import comments relative to imports.